src/Application/Components/Shop/Shop.jsx

```diff
@@ -27,10 +32,12 @@ const Shop = React.createClass({
                   owned={asset.owned}
                   keyCode={assetDefaults[asset.name].keyCode}
                   onClick={_.partial(this.props.onItemClick, asset.id)}
+                  areStatsVisible={this.props.areStatsVisible}
                 />
               );
             })}
           </div>
+
```

Comment:
ah i see there's newlines all over the place in aaron's existing code.  if you want to chop up a long html block more, extract chunks of it out to their own `renderSomething()` methods